engine/engine/src/test/wscript

```diff
@@ -91,10 +91,24 @@ def build(bld):
         bob_flags.append("--use-vanilla-lua")
     bob_flags = ' '.join(bob_flags)
 
-    # Build engine test content using bob
-    # We always run distclean. Hook into waf clean/distclean in some way?
-    bob_command = 'java -jar ./build/content/bob-engine.jar -root ./src/test -k clean build %s' % bob_flags
+    # Clean folder
```

Comment:
this custom logic is needed to build custom files linked from game.project which can't be build the other way.
so we 
* clean folder forst
* then build custom ini files projects
* then build the main game.project  (it should be the last one to make sure the right collection is in `game.proejctc`)